src/create-render-context.js

```diff
@@ -172,6 +177,21 @@ const patchTimers = (): void => {
     patchCallbackFnWithGate(window, "requestAnimationFrame", "__SSR_ACTIVE__");
 };
 
+const createVirtualConsole = () => {
+    const virtualConsole = new VirtualConsole();
+    virtualConsole.on("jsdomError", (e: Error) => {
+        if (e.message.indexOf("Could not load img") >= 0) {
+            // We know that images cannot load. We're deliberately blocking
+            // them.
+            return;
+        }
+        // eslint-disable-next-line no-console
+        console.error(`JSDOM: ${e.stack}`);
+    });
+    virtualConsole.sendTo(console, {omitJSDOMErrors: true});
```

Comment:
@jeresig and he we just tell the console to map everything else to the node console still, but keep the jsdom ones to our special handler.